sru-templates/manual/azure-sru

```diff
@@ -15,11 +17,42 @@ RESOURCE_GROUP="<specify your resource group>"
 BOOT_DIAG="<specify your boot diagnostics group>"
 SSH_KEY="$HOME/.ssh/id_rsa.pub"
 
+
+IPV6_NET_REGISTERED=`az feature show --name AllowIPv6VirtualNetwork --namespace Microsoft.Network | jq -r '.properties.state'`
+IPV6_CA_LB_REGISTERED=`az feature show --name AllowIPv6CAOnStandardLB --namespace Microsoft.Network | jq -r '.properties.state'`
+
+if [ "$IPV6_NET_REGISTERED" != "Registered" -o "$IPV6_CA_LB_REGISTERED" != "Registered" ]; then
+	echo "Account is not yet registered for AllowIPv6VirtualNetwork or AllowIPv6CAOnStandardLB. Can take 30 mins. Activate feature now (y/n)?"
+    read RESP
+    if [ "$RESP" = "y" ]; then
+        az feature register --name AllowIPv6VirtualNetwork --namespace Microsoft.Network
+        az feature register --name AllowIPv6CAOnStandardLB --namespace Microsoft.Network
+    fi
```

Comment:
If the response is anything other than `y` here, I think we'll end up in an infinite loop below, right?

Reply:
@OddBloke the intent is infinite until azure cli confirms that the account is properly registered with ipv6 feature (which took about 20 mins when I set my account up)

Reply:
But if $RESP is, say "n", then we won't kick off the feature registration but we _will_ loop waiting for it to complete.

Reply:
@OddBloke the request for feature registration only happens once non-blocking, then the backend takes 20 mins to process. Re-requests are not required if the cli came back 0 on the initial feature register call.

Reply:
OK, perhaps I'm not expressing myself properly here. The user will be prompted with:

> Account is not yet registered for AllowIPv6VirtualNetwork or AllowIPv6CAOnStandardLB. Can take 30 mins. Activate feature now (y/n)?

If they answer "n" here, then $RESP will be "n". In that case, we will _not_ call `az feature register` at all (because `"$RESP" = "y"` is not true). However, we _will_ then proceed to echo:

> Waiting for AllowIPv6VirtualNetwork feature registration

and enter the `while [ "$IPV6_NET_REGISTERED" != "Registered" -o "$IPV6_CA_LB_REGISTERED" != "Registered" ]` loop. However, because we didn't call `az feature register` at all, we are never going to see those values switch to "Registered", and therefore we are never going to exit this while loop.

Reply:
ohh! I'm being thick. you are correct. and yes that would be the case. The only reason to opt out would be to make sure the check/looping  on feature does not proceed if the feature is disabled. This was only needed during my development of that ipv6 feature. But, let's make it work in case we decide to start enabling/disabling that feature on various manual runs in the future.

Reply:
Added an option to exit or avoid performing advanced network tests if feature is not registered